app/Http/Controllers/Three/PostsController.php

```diff
@@ -149,9 +149,13 @@ public function show(Post $post)
      */
     public function update(PostRequest $request, Post $post)
     {
-        $post->update($request->only('status', 'caption'));
+        if (token()->role() === 'admin') {
```

Comment:
Do we want to allow a user to update their own post? (If `auth()->id() == $post->northstar_id`?)

Reply:
oh I didn't think about that! I don't think we have UI for that now but definitely could see that in the future - I'll add this!